<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QDialog" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#optionsFrame {
border: none;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);
}

#bottomFrame {
border: none;
background:  white;
}

#optionsFrame QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 70px;
}

#optionsFrame QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

#optionsFrame QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="optionsFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="graphComboBox">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>5</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Raster plot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Current graph</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Local field potential</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Phase portrait plot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mean Firing rate</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="printButton">
      <property name="geometry">
       <rect>
        <x>156</x>
        <y>4</y>
        <width>84</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="channelsSpinBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>4</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Number of channels in this block (neural ensemble).</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="depthSpinBox">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>4</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The depth of the firing rates.</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="updateRateSpinBox">
      <property name="geometry">
       <rect>
        <x>421</x>
        <y>4</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Update rate for the calculation of the firing rates.</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="rescaleButton">
      <property name="geometry">
       <rect>
        <x>467</x>
        <y>4</y>
        <width>84</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Rescale</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="lineThicknessSpinBox">
      <property name="geometry">
       <rect>
        <x>554</x>
        <y>4</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Line thickness.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>243</x>
        <y>4</y>
        <width>84</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="verticalCheckBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>4</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottomFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QFrame" name="plotFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="plotLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
